sandbox-backend/src/main/java/com/exadel/sandbox/team5/dao/TagDAO.java

```diff
@@ -1,8 +1,16 @@
 package com.exadel.sandbox.team5.dao;
 
 import com.exadel.sandbox.team5.entity.Tag;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface TagDAO extends CommonRepository<Tag> {
+
+    @Query(value = """
+            SELECT c.id
+            FROM sandbox.tag c
```

Comment:
`sandbox` using of schema is not required

Reply:
Done.